test/validation/ValidatedActionsSpec.scala

```diff
@@ -103,7 +83,7 @@ class ValidatedActionsSpec extends FrontEndTestHelper {
     "return a redirect to the transfer agreement page if the transfer agreement is not complete" in {
       val consignmentId = UUID.randomUUID()
 
-      val graphqlConfigurationMock = mockUploadPermittedGraphqlResponse(itac.Data(Option(itac.GetTransferAgreement(false))), gcs.Data(Option(GetConsignment(CurrentStatus(Option.empty)))))
+      val graphqlConfigurationMock = mockUploadPermittedGraphqlResponse(itac.Data(Option(itac.GetTransferAgreement(false))), gcs.Data(Option(GetConsignment(CurrentStatus(None, Option.empty)))))
```

Comment:
Is it significant that one of these parameters is `None` and one is `Option.empty`? If not, it would be less confusing for them to be the same.

Reply:
👍  I'm not entirely sure why the upload status is `Option.empty` but yes, I agree, it should be `None` for both